addons/point_of_sale/models/account_move.py

```diff
@@ -39,7 +39,7 @@ def _get_invoiced_lot_values(self):
                             'quantity': line.qty if lot.product_id.tracking == 'lot' else 1.0,
                             'uom_name': line.product_uom_id.name,
                             'lot_name': lot.lot_name,
-                            'lot_id': lot.id,
```

Comment:
why not did you keep it and made the search to locate the right id 

something like here
https://github.com/odoo/odoo/blob/2bf71a32000c33754a32599776079a0dc60d2bba/addons/point_of_sale/models/stock_picking.py#L215-L220

Reply:
As far as i can tell that 'lot_id' key is not used in standard code, so such a mapping would add unnecessary complexity. Especially in stable.

Reply:
you right, let's move it then to another key